test/integration_test/README.md

```diff
@@ -1,36 +1,36 @@
 ## Integration Test
 
-### Preparation
+## Preparation
 
 1. Install Chaos Mesh
 
-You can install Chaos Mesh by commands:
+   You can install Chaos Mesh by commands:
 
-```bash
-hack/local-up-chaos-mesh.sh
-kubectl set env deployment/chaos-dashboard SECURITY_MODE=true -n chaos-mesh
-kubectl port-forward -n chaos-mesh svc/chaos-dashboard 2333:2333 &
-```
+   ```bash
+   hack/local-up-chaos-mesh.sh
```

Comment:
I think this line of doc could be replaced by

```bash
make image
minikube start
minikube image load ghcr.io/chaos-mesh/chaos-daemon:latest
minikube image load ghcr.io/chaos-mesh/chaos-mesh:latest
minikube image load ghcr.io/chaos-mesh/chaos-dashboard:latest
helm upgrade --install chaos-mesh ./helm/chaos-mesh/ -n chaos-mesh --create-namespace --set dashboard.securityMode=false
```

I am not sure that's a drop in replacement 🤔 maybe it requires some tunning work.

Reply:
I still use the kind for bootstrapping the cluster as before in CI. Yes, this document needs some "tunning work" to fit the current development environment.

Reply:
Prepare to replace with https://chaos-mesh.org/docs/configure-development-environment/.